### PR TITLE
📓 Updated HUN

### PR DESCRIPTION
**Pull request by KristofKekesi**

### CODE REVIEW

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 1]
```diff
@@ -2280,53 +2280,53 @@
...
         </trans-unit>
         <trans-unit id="DriveTypeRam" translate="yes" xml:space="preserve">
           <source>RAM Disk Drive</source>
-          <target state="new">RAM Disk Drive</target>
+          <target state="translated">RAM meghajtó</target>
         </trans-unit>
         <trans-unit id="DriveTypeRemovable" translate="yes" xml:space="preserve">
           <source>Removable Storage Device</source>
-          <target state="new">Removable Storage Device</target>
+          <target state="translated">Eltávolítható meghajtó</target>
```
Comment (KristofKekesi) [inline Files/MultilingualResources/Files.hu-HU.xlf:2311]: @nvi9 Is there a more suitable translation?

Comment (nvi9) [reply Files/MultilingualResources/Files.hu-HU.xlf:2311]: I think it's good enough, see my initial translations here: 89402daa

Comment (KristofKekesi) [reply Files/MultilingualResources/Files.hu-HU.xlf:2311]: Oh I didn't see there is a commit for this.

Comment (KristofKekesi) [reply Files/MultilingualResources/Files.hu-HU.xlf:2311]: I searched what an unmounted drive is. It is a drive without a drive letter. So I would translate it ```bjektumnév nélküli helyi lemez``` not ```Nem csatlakoztatott meghajtó```.

**Review by yaichenbaum — APPROVED** [review]
:shipit:

**Pull request merged**